setup.cfg

```diff
@@ -107,28 +107,28 @@ optional =
     triangle
 testing =
     babel>=2.9.0
-    pytest>=7.0.0
-    pytest-qt
-    hypothesis>=6.8.0
-    xarray
     fsspec
-    zarr
+    hypothesis>=6.8.0
+    lxml
     matplotlib
     pooch>=1.3.0
-    semgrep
-    tensorstore>=0.1.13 ; python_version < '3.10'
-    torch>=1.7 ; python_version < '3.10'
-    lxml!=4.9.0
+    pytest-cov
+    pytest-qt
+    pytest>=7.0.0
+    tensorstore>=0.1.13
+    torch>=1.7
```

Comment:
Given that the 3.10 tests pass, I guess the environment markers for `tensorstore` and `torch` are no longer needed?

Reply:
yeah, those were because wheels weren't yet available when we added py3.10 support: https://github.com/napari/napari/pull/3702